CMakeLists.txt

```
@@ -129,7 +129,7 @@ configure_package_config_file(
 
 configure_file(cmake/BackwardConfig.cmake ${CMAKE_CURRENT_BINARY_DIR}/BackwardConfig.cmake @ONLY)
 
```

```suggestion

install(FILES ${CMAKE_CURRENT_BINARY_DIR}/BackwardConfig.cmake DESTINATION share/${PROJECT_NAME}/cmake)
```
Let's add the previous one too just in case. So, we don't disturb the original configuration. In my testing, it seems to work. If this is fine for you, please accept it and then I will merge it